chandra_aca/aca_image.py

```diff
@@ -266,3 +304,94 @@ def centroid_fm(img, bgd=None, pix_zero_loc='center', norm_clip=None):
         raise ValueError("pix_zero_loc can be only 'edge' or 'center'")
 
     return row, col, norm
+
+
+class AcaPsfLibrary(object):
+    """
+    Access the ACA PSF library, whch is a library of 8x8 images providing the integrated
+    (pixelated) ACA PSF over a grid of subpixel locations.
+
+    :param filename: file name of ACA PSF library (default=built-in file)
+    :returns: AcaPsfLibrary object
+    """
+
+    def __init__(self, filename=None):
+        from astropy.table import Table  # Table is a somewhat-heavy import
```

Comment:
Did we decide on a convention of putting a comment or something near the top for non-top imports?

Reply:
We talked about this but didn't make a decision.  Probably not going to happen with enough consistency to make it worthwhile.